### PR TITLE
enable powertools for running plugin tests on almalinux

### PR DESCRIPTION
**Pull request by d-w-moore**
https://github.com/irods/irods_netcdf/issues/25 is related but this requirement may apply in a broader way whenever a plugin depends on system libraries (ie build and runtime prerequisites other than irods-externals)

### CODE REVIEW

File: projects/almalinux-8/Dockerfile
Comment (alanking) [inline projects/almalinux-8/Dockerfile:56]: Would it be possible to add these to the build step which begins on line 3 to avoid adding the two additional layers?

Comment (alanking) [reply projects/almalinux-8/Dockerfile:56]: Or, if you could combine these two `RUN` commands into one that would be fine, too.

Comment (alanking) [reply projects/almalinux-8/Dockerfile:56]: @d-w-moore *poke*

Comment (alanking) [reply projects/almalinux-8/Dockerfile:52]: Done

**Comment by alanking** [comment]
I rebased the changes and added a commit to avoid the extra layers. I confirmed that I can stand up almalinux:8 with these changes, but haven't tried running any tests which require the powertools.

Will await confirmation from @d-w-moore and approval before merging.

**Review by korydraughn — COMMENTED** [review]
Changes look fine.

Assuming core and unit tests pass, I don't see a reason to not accept this.

Will await @d-w-moore's confirmation before approving.

**Comment by alanking** [comment]
Unit tests passed for 4.3.0 as released. Core tests are running now.

**Comment by alanking** [comment]
Core tests passed for 4.3.0 as released.

**Review by korydraughn — COMMENTED** [review]
Add the pounds when ready.

**Review by korydraughn — APPROVED** [review]

**Pull request merged**
